runtime/ftdetect/tutor.vim

```diff
@@ -0,0 +1 @@
+autocmd! BufRead,BufNewFile *.tutor setfiletype tutor
```

Comment:
:+1: Thanks for pointing this out, it's really easy to forget we are "not in Kansas anymore".